src/oaklib/io/streaming_fhir_writer.py

```diff
@@ -24,10 +24,31 @@ def emit_multiple(self, entities: Iterable[CURIE], **kwargs):
             g = oi.extract_graph(list(entities), include_metadata=True)
             gd = GraphDocument(graphs=[g])
             logging.info(f"Converting {len(g.nodes)} nodes to OBO")
-            converter = OboGraphToFHIRConverter()
+            converter = OboGraphToFhirJsonConverter()
             converter.curie_converter = oi.converter
             code_system = converter.convert(gd)
             logging.info(f"Writing {len(code_system.concept)} Concepts")
+            # TODO: Should not this call OboGraphToFhirJsonConverter.dump()?
             self.file.write(json_dumper.dumps(code_system))
         else:
             super().emit_multiple(entities, **kwargs)
+
+
+# TODO:
+@dataclass
+class StreamingFhirNpmWriter(StreamingWriter):
```

Comment:
@cmungall I didn't even notice these existed until a short time ago. Do you need me to implement `StreamingFhirNpmWriter`? Does it make sense to?